test_runner/fixtures/utils.py

```diff
@@ -155,7 +155,7 @@ def get_scale_for_db(size_mb: int) -> int:
 
 
 ATTACHMENT_NAME_REGEX = re.compile(
-    r".+\.log|.+\.stderr|.+\.stdout|.+\.filediff|.+\.metrics|flamegraph\.svg|regression\.diffs"
+    r".+\.log|.+\.stderr|.+\.stdout|.+\.filediff|.+\.metrics|flamegraph\.svg|regression\.diffs|.+\.html"
```

Comment:
To support this properly in Allure we also need to add to `allure_attach_from_dir` below:

```python
elif source.endswith(".html"):
    attachment_type = "text/html"
    extension = "html"
```